src/suites/cts/copyImageBitmapToTexture.spec.ts

```diff
@@ -0,0 +1,83 @@
+export const description = `
+copy imageBitmap To texture tests.
+`;
+
+import { TestGroup } from '../../framework/index.js';
+
+import { GPUTest } from './gpu_test.js';
+
+export const g = new TestGroup(GPUTest);
+
+function calculateRowPitchForImageBitmap(width: number): number {
+  if (width % 256 === 0) {
+    return 4 * width;
+  }
+  return 4 * (width + 256 - (width % 256));
+}
```

Comment:
IIRC the pitch needs to be a multiple of 256 bytes, not 256 pixels (correct me if I'm wrong).

Reply:
You're right. Done.